cpp/include/cudf/lists/lists_column_view.hpp

```diff
@@ -88,6 +107,8 @@ class lists_column_view : private column_view {
    * on lists columns should be using `get_sliced_child()` instead of `child()`.
    *
    * @throw cudf::logic error if this is an empty column
+   * @param stream CUDA stream used for device memory operations and kernel launches
```

Comment:
This is another parameter that it would be nice to have a standardized docstring for (similar to my request for Nullate in another one of your PRs). It would also help us be consistent in documenting this object as an RMM wrapper for a CUDA stream as opposed to a raw cudaStream_t.

Reply:
Long back, I proposed to have the parameters of `stream` and `mr` documented at a single file, and use `@copydetails` to copy it. It does not give good experience of reading docs in the source code. https://github.com/rapidsai/cudf/pull/10964#discussion_r882035940 clarifies this.

Reply:
Yup I'm fine with skipping this after that discussion, thanks.